react-app/src/Login.js

```diff
@@ -0,0 +1,52 @@
+
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Login extends Component {
+    state = { 
+        username: '',
+        password: '',
+    };
+
+    render() { 
+        return ( 
+            <form onSubmit={this.handleSubmit}>
+                <label htmlFor="username">username</label>
+                <input type="text"
+                    name="username"
+                    onChange={this.handleInputChange}
+                    value={this.state.username}
+                />
+                <label htmlFor="password">password</label>
+                <input type="password"
+                    name="password"
+                    onChange={this.handleInputChange}
+                    value={this.state.password}
```

Comment:
don't forget to add the `required` attribute on any field you want to make sure never gets submitted as an empty string